packages/blade/src/components/Typography/BaseText/BaseText.tsx

```diff
@@ -18,21 +55,25 @@ const BaseText = ({
   ...rest
 }: BaseTextProps): ReactElement => {
   const { theme } = useTheme();
-  const textColor = getIn(theme.colors, color);
+  const textColor: string = getIn(theme.colors, color, '');
+  const themeFontfamily = theme.typography.fonts.family[fontFamily];
+  const themeFontSize = makeTypographySize(theme.typography.fonts.size[fontSize]);
+  const themeFontWeight = theme.typography.fonts.weight[fontWeight];
+  const themeLineHeight = makeTypographySize(theme.typography.lineHeights[lineHeight]);
```

Comment:
We are skipping the runtime checks for these theme accesses?

Reply:
nupe actually I forgot to push 🤦‍♂️